### DISCUSSION TITLE
centroidal

### DISCUSSION THREAD

**Discussion by Mulike-568** [neperfepx/neper#604] [Q&A] [answered]
Hello there!
I would like to output the centroidal of each surface of this model.
Now I'm testing the` neper -T  -n  20  -dim  2   -format  geo   -morpho 'diameq:lognormal(20,1),sphericity:normal(0.9,0.02)'`
Indeed, I want to know if it is possible for me to get the point of centroidal so that i can export it into gmsh and perform further analysis.
best regards
WX

**Comment by rquey** [comment, marked as answer]
Use `-statcell x,y`. For some reasons, you have to write the tess file for the stat file to print (small bug). So, your command becomes:

```
neper -T -n 20 -dim 2 -format tess,geo -morpho 'diameq:lognormal(20,1),sphericity:normal(0.9,0.02) -statcell x,y
```
